src/Uno.UI/UI/Xaml/UIElement.Layout.crossruntime.cs

```diff
@@ -149,11 +149,6 @@ public void Measure(Size availableSize)
 				return; // Only FrameworkElements are measurable
 			}
 
-			if (double.IsNaN(availableSize.Width) || double.IsNaN(availableSize.Height))
-			{
-				throw new InvalidOperationException($"Cannot measure [{GetType()}] with NaN");
-			}
```

Comment:
That was originally placed because UWP refused to measure with NaN. Won't this hide other bugs?

Reply:
@jeromelaban When I tested in WinUi, it was interesting as I came across two cases:

1. C# code tries to measure with NaN. This fails with COMException.
2. C# code that causes an internal measure with NaN where the "Measure" call happens in the C++ code of WinUI. This does **not** fail.

We can't replicate that behavior, and the fix for ContentControl revealed the case "2" where it works in WinUI and throws in our case.